dist/index.html

```diff
@@ -131,22 +140,73 @@ <h2 id="faq">FAQ</h2>
 
     //Render icon-grid
     document.querySelector('.docs-icons').innerHTML = Object.keys(groups).map((type) => `
-      <h3 class="nrk-xs-12of12">${type}<br>&nbsp;</h3>
+      <h3 class="nrk-xs-12of12" style="color:#26292a">${type}<br>&nbsp;</h3>
```

Comment:
Would have preferred to extract the styles out using classNames instead of having inline rules, i get the gist.
However it makes reading the template strings easier.